tests/BusinessTest.php

```diff
@@ -494,6 +494,31 @@ public function testSerializeUnserialize()
         );
     }
 
+    public function testJsonSerialize()
+    {
+        $holiday1 = new \DateTime('2015-05-11');
+        $holiday2 = new \DateTime('2015-05-12');
+
+        $business = new Business(
+            [
+                new Day(Days::MONDAY, [['09:00', '13:00'], ['14:00', '17:00']]),
+                new SpecialDay(
+                    Days::FRIDAY,
+                    function (\DateTime $date) {
+                        return [['10:00', '13:00'], ['14:00', '17:00']];
+                    }
+                ),
+            ],
+            new Holidays([$holiday1, $holiday2]),
```

Comment:
Could you also use a `DateRange` as holiday please (as shown on the README example) ?